client/my-sites/plans/woo-express-plans-page/index.tsx

```diff
@@ -119,66 +118,66 @@ const WooExpressPlansPage = ( {
 		} );
 	}, [] );
 
-	const planFeatures =
-		isEnabled( 'plans/wooexpress-small' ) || isWooExpressSmallPlan( currentPlan.productSlug ) ? (
-			<WooExpressPlans
-				interval={ interval ? interval : planInterval }
-				monthlyControlProps={ { path: plansLink( '/plans', selectedSite.slug, 'monthly', true ) } }
-				siteId={ selectedSite.ID }
-				siteSlug={ selectedSite.slug }
-				triggerTracksEvent={ triggerPlansGridTracksEvent }
-				yearlyControlProps={ { path: plansLink( '/plans', selectedSite.slug, 'yearly', true ) } }
-				showIntervalToggle={ showIntervalToggle }
-			/>
-		) : (
-			<>
-				<h2 className="woo-express-plans-page__section-title">
-					{ translate( 'What’s included in %(planName)s plan', {
-						args: {
-							planName: activePlan.getTitle(),
-						},
-					} ) }
-				</h2>
-
-				<div className="woo-express-plans-page__features-wrapper">
-					{ wooExpressMediumPlanFeatureSets.map( ( featureSet ) => (
-						<TrialFeatureCard key={ featureSet.title } { ...featureSet } />
-					) ) }
+	const planFeatures = isWooExpressSmallPlan( currentPlan.productSlug ) ? (
```

Comment:
This should now be `true` in all cases, not only for the small Woo Express plan. That exception was for the case where the user had purchased a small plan already, but the feature flag wasn't enabled. Now that the feature flag is enabled, the original condition should always evaluate to true.
```suggestion
	const planFeatures = (
```